specification/cognitiveservices/data-plane/Speech/SpeakerRecognition/identification/identification-swagger.json

```diff
@@ -0,0 +1,771 @@
+          "description": "A float number indicating the similarity between input audio and targeted voice print. This number must be between 0 and 1. A higher number means higher similarity."
+        }
+      }
+    },
+    "Error": {
+      "required": [
+        "error"
+      ],
+      "properties": {
+        "error": {
+          "properties": {
+            "code": {
+              "type": "string"
+            },
+            "message": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "NotFound": {
+      "description": "NotFound - Requested profile doesn't exist",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      },
+      "examples": {
+        "application/json": {
+          "code": "Not Found",
+          "message": "Requested profile doesn't exist"
+        }
+      }
+    },
+    "Unauthorized": {
+      "description": "Request is not authorized. Make sure subscription key is included and valid.",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      },
+      "examples": {
+        "application/json": {
+          "code": "Unauthorized",
+          "message": "Request is not authorized. Make sure subscription key is included and valid."
+        }
+      }
+    },
+    "RateLimit": {
+      "description": "Rate limit is exceeded.",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      },
+      "examples": {
+        "application/json": {
+          "code": "RateLimit",
+          "message": "Rate limit is exceeded."
+        }
+      }
+    },
+    "ServerError": {
+      "description": "Internal Server Error.",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      },
+      "examples": {
+        "application/json": {
+          "code": "InternalServerError",
+          "message": "Internal Server Error."
+        }
+      }
+    }
+  },
+  "paths": {
+    "/text-independent/profiles": {
+      "post": {
+        "description": "Creates a new speaker profile with specified locale.",
+        "summary": "Create Profile",
+        "tags": [
+          "text-independent"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "profileInfo",
+            "description": "Provide following detail info when creating a new profile.\nFields  | Description\n------- | ------------\nlocale  | Locale for the language of this speaker profile. A complete supported locale list is here: <ul><li>**en-US (American English)**</li><li>**es-ES (Castilian Spanish)**</li><li>**fr-FR (Standard French)**</li><li>**zh-CN (Mandarin Chinese)**</li></ul>",
+            "schema": {
+              "required": [
+                "locale"
+              ],
+              "properties": {
+                "locale": {
+                  "type": "string",
+                  "example": "en-us"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
```

Comment:
@HeidiHanZhang -- Check to make sure that your operations have a default response specified.

Reply:
done